packages/appium/docs/en/developing/documents-translation.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Does https://github.com/appium/appium/blob/master/packages/appium/docs/mkdocs-en.yml need to have this file reference?

Reply:
I would instead suggest adding the contents of this file to `contributing.md`, as a new level-2 section. I don't think this has much to do with development.

Reply:
done